src/components/Todo/Todo.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import { User } from '../User/User';
+import { TodoWithUsers } from '../../types';
+import './Todo.css';
+
+interface Props {
+  todo: TodoWithUsers;
+}
+
+export const Todo: FC<Props> = ({ todo: { title, user, completed } }) => {
+  return (
```

Comment:
не очень удобная деструктуризация
```suggestion
export const Todo: FC<Props> = ({ todo }) => {
  const { title, user, completed } = todo;
  
  return (
```

Reply:
Ок